certvalidator/validate.py

```diff
@@ -1447,7 +1454,10 @@ def verify_crl(cert, path, validation_context, use_deltas=True, cert_description
                 certificate_list
             ))
             continue
-        if moment > certificate_list['tbs_cert_list']['next_update'].native:
+
+        # according to RFC 5280 5.1.2.5 next_update is optional
+        if certificate_list['tbs_cert_list']['next_update'] and \
+           moment > certificate_list['tbs_cert_list']['next_update'].native:
```

Comment:
Upon further reflection, I don't believe that this should be included in this PR because it has nothing to do with moment. I also seem to have lost the reference to the file I was using that exercised this condition for testing.